scripts/chroot.sh

```diff
@@ -13,16 +13,23 @@ export LIVE_BOOT_SCRIPTS="casper lupin-casper"
 dpkg-divert --local --rename --add /sbin/initctl
 ln -s /bin/true /sbin/initctl
 
+# Installing wget
+apt-get install wget apt-transport-https
```

Comment:
@vaibhavsingh97 the installed packages which are not part of distribution but were installed inside chroot for some purpose should be removed in cleanup part that you can find at the end of chroot script